contests/terms.html

```diff
@@ -0,0 +1,239 @@
+      <li>
+        本⼤会へは個⼈でご応募いただきます。
+        応募するにあたり、本規約に同意した上で応募するものとします。
+      </li>
+      <li>親権者の同意（本規約の同意も含む）を得た上で応募するものとします。</li>
+
+      <li>
+        応募者が以下の事項に該当すると主催者が判断した場合、主催者は、応募者に予告なく当該応募を無効とし、また⼊賞後であっても⼊賞等の取消をし、且つ賞⾦・賞品等の返還等を求める場合があります。
+        <ol>
+          <li>本規約に定める応募資格を満たしていない場合</li>
+          <li>応募内容に不備がある場合</li>
+          <li>応募内容の不備についての主催者からの通知に対し、3⽇（通知内に期限が明記されている場合はそちらを優先）以内に返答がない場合</li>
+          <li>作品が第４条の遵守事項⼜は本規約のいずれかに違反している場合</li>
+          <li>本⼤会への参加が困難であると認められる場合</li>
+          <li>その他主催者が応募者の本⼤会参加が不適当と判断する場合</li>
+        </ol>
+      </li>
+
+      <li>
+        応募する作品は、以下の条件を満たしたものとします。
+        <ol>
+          <li>応募者本⼈または応募者を含むチームが作成した作品であること</li>
+          <li>チームの場合はチームメンバー全員が第1条4項(2)(3)を満たしていること</li>
+          <li>正常に動作する作品であること</li>
+          <li>暴⼒的、差別的な内容を盛り込んでいない作品であること</li>
+          <li>応募する作品は、コンテスト前に作成したもの、本⼤会以外で応募されたものでも応募できるものとします。</li>
+        </ol>
+      </li>
+
+      <li>応募にはProtoPediaサービスを利用するため、ProtoPediaアカウントは親権者の同意を得て作成することを含め <a href="https://protopedia.net/terms" target="_blank">ProtoPedia利用規約</a> にも全て同意した上で応募するものとします。</li>
+      <li>ProtoPediaで公開した情報は公開範囲によりインターネット上で閲覧できる状態になり、記載された文書・画像の情報はDojoConでの様々な広報で使用することがあることに同意したものとします。</li>
+    </ol>
+
+    <h3>第３条 （応募情報の取り扱い）</h3>
+
+    <p>
+      応募情報についてはDojoCon Japan実⾏委員会が取り扱います。
+      ご⼊⼒いただいた応募者の個⼈情報は下記の通り取り扱います。
+    </p>
+
+    <ol>
+      <li>
+        応募者の個⼈情報は以下の⽬的で利⽤します。
+        <ol>
+          <li>本⼤会へ参加される応募者の申し込み受付、及び出⽋の確認。</li>
+          <li>応募者からの質問、お問い合わせへの対応。</li>
+          <li>コンテストの審査、審査結果の連絡。</li>
+          <li>応募者の作品へ対する質問などの連絡。</li>
+        </ol>
+      </li>
+
+      <li>応募者の同意がある場合、または法令に基づく場合を除き、主催者以外の第三者に個⼈情報を提供いたしません。</li>
+      <li>個⼈情報はGoogle社サービスを利⽤して管理いたします。</li>
+      <li>個⼈情報をご提供いただけない場合は、本⼤会へ参加できないことをご了承願います。</li>
+      <li>個⼈情報取り扱い責任者：DojoCon Japan 実⾏委員⻑</li>
+      <li>
+        個⼈情報に関するご相談、開⽰等に関するお問い合わせは以下にご連絡ください。 <br>
+        <a href="https://forms.gle/t9ctDcuZZ8cqqFtR6" target="_blank">DojoCon Japan 2025 お問い合わせフォーム</a>
+      </li>
+    </ol>
+
+    <h2>第４条 （遵守事項）</h2>
+
+    <ol>
+      <li>本⼤会において応募者は、スタッフの注意等に従うものとし、本⼤会の円滑な運営に協⼒するものとします。</li>
+
+      <li>
+        応募者は、本⼤会において、⾃らが独⾃に作成した作品だけを応募するものとし、また、以下各号に該当するような作品を応募してはならないものとします。
+        なお、これらの作品が応募等された場合、主催者は応募者の承諾を得ることなく、当該作品を選考から除外いたします。
+        <ol>
+          <li>主催者、その他第三者の肖像権、パブリシティ権、著作権、商標権、特許権等の知的財産権その他の権利を侵害する⼜はそのおそれのあるもの</li>
+          <li>わいせつな表現・内容を含むもの</li>
+          <li>法令に違反する⼜はそのおそれのあるもの</li>
+          <li>公序良俗に反する表現・内容を含む作品⼜はそのおそれのあるもの</li>
+          <li>その他主催者が不適当と判断するもの</li>
+        </ol>
+      </li>
+
+      <li>作品に関して第三者（原作者、著作者等を含み、これらに限定されません）からの権利の主張、異議、苦情、損害賠償請求等の紛争が⽣じた場合には、応募者の責任と費⽤負担（弁護⼠費⽤等をも含む）においてこれを処理し、主催者に損害を与えないものとします。</li>
+    </ol>
+
+    <h2>第５条 （権利帰属）</h2>
+
+    <ol>
+      <li>
+        作品にかかる著作権その他の知的財産権の帰属は、以下の通りとします。
+        <ol>
+          <li>応募者が著作権保持者から作品制作のために提供を受けた商標やロゴ、画像、プログラム（Android及びiOS等を含む）、データベース、キャラクターその他素材に関する権利は、当該著作権保持者に帰属します。</li>
```

Comment:
`（Android及びiOS等を含む）`
の意図を確認しておきたいかもと思いました

Reply:
こちらも削除でお願いします
https://discord.com/channels/1355061393992323192/1357695215594504485/1396824707348828347
```suggestion
          <li>応募者が著作権保持者から作品制作のために提供を受けた商標やロゴ、画像、プログラム、データベース、キャラクターその他素材に関する権利は、当該著作権保持者に帰属します。</li>
```